src/wave.ts

```diff
@@ -8,6 +8,7 @@ export class Wave {
     private center?: TextEditorDecorationType;
     private colors: string[] = [];
     private amplitude: number = 3;
+    private fontWeight: string = "900";
```

Comment:
Can you single quote this? I need to add some linting rules.

Reply:
For sure!

Reply:
Updated to single quotes :)